drivers/include/adxl345.h

```diff
@@ -159,7 +159,7 @@ typedef struct {
 typedef struct {
     adxl345_params_t *params;       /**< Device configuration */
     adxl345_interrupt_t interrupt;  /**< Interrupts configuration  */
-    float scale_factor;             /**< Scale factor for converting value to mg */
+    int16_t scale_factor;           /**< Scale factor for converting value to mg */
```

Comment:
This must also stay in float for now.

Reply:
"Must"?

Ah, screw it. Fix you shitcode by yourself.

All the more reasons to maintain own fork instead of participating in nonsense like this.

Reply:
Well, I think your reaction is not appropriate and could be considered as a violation of your [code of conduct](https://github.com/RIOT-OS/RIOT/blob/master/CODE_OF_CONDUCT.md). Again, please calm down and keep this kind of comment for yourself.
I (and I'm sure others as well) appreciate your interest in RIOT and the effort you are putting to make it better.

Reply:
@aabadie otherwise what?

You'll never accept my PR again and will stay with shitcode like this? Take a look at my last commit here, it's absolutely marvellous.

BTW, is there a way to list all commits made by @dylad? I'm dead serious to check them all or at least mark as potentially dangerous in our fork. That's not production code, that's not even beta code, that's shamefull code - so badly its written.

Reply:
@olegart of course there is, did you try reading the fine manual for `git log`? https://git-scm.com/docs/git-log#git-log---authorltpatterngt Do whatever you like with that info.

Reply:
Oh, I know exactly what to do with that info.

With that info I'll return in a year to see if someone has bothered to fix the shitcode here.

I'm kind of know what I see then, but I'm pretty sure Code of Conduct will be vastly improved.

Reply:
@dylad 
I'm kinda curious, why it _must_ stay float and _for how long_?
It's clearly not about precision, driver was not able to show 1g correctly.
Is it for backwards compatibility?